examples/wallets/src/lib.rs

```diff
@@ -191,33 +191,37 @@ mod tests {
 
     #[tokio::test]
     #[allow(unused_variables)]
-    async fn setup_wallet_custom_assets() -> Result<(), Box<dyn std::error::Error>> {
+    async fn setup_wallet_custom_assets() -> Result<(), rand::Error> {
         // ANCHOR: custom_assets_wallet
         use fuels::prelude::*;
         use rand::Fill;
 
         let mut wallet = LocalWallet::new_random(None);
         let mut rng = rand::thread_rng();
 
-        let asset_id_base = BASE_ASSET_ID;
-        let coins_per_asset_base = 2;
-        let amount_per_coin_base = 4;
+        let asset_base = AssetConfig {
```

Comment:
`AssetConfig` is indeed much cleaner, like it!

Reply:
Thanks!